frontend/src/modules/lf/segments/components/view/lf-project-groups-table.vue

```diff
@@ -1,72 +1,46 @@
 <template>
-  <div class="mb-6 h-10 flex items-center">
-    <app-pagination-sorter
-      :page-size="pagination.pageSize"
-      :total="pagination.total"
-      :current-page="pagination.currentPage"
-      :has-page-counter="false"
-      position="top"
-      module="project group"
-      @change-sorter="onPageSizeChange"
-    />
-  </div>
-
-  <div class="app-list-table not-clickable panel !px-0 !pt-0 !pb-6">
+  <div class="app-list-table not-clickable !px-0 !pt-0 !pb-6">
     <el-table
       id="project-groups-table"
       ref="table"
-      :data="list"
+      :data="projectGroups.list"
       row-key="id"
       :resizable="false"
+      @row-click="handleRowClick"
     >
-      <!-- Name -->
+      <!-- Status -->
       <el-table-column
-        label="Project Group"
-        prop="name"
-        width="300"
-        fixed
+        label="Status"
```

Comment:
The column headers are not matching the designs:
They are with 12px instead of 13px and the spacing around is bigger than it should be.
I think here either we override the element-plus styling for the cells header, or we try to use the table elements from the ui-kit which should offer more flexibility

Reply:
I'll override the styling for now. The current implementation of the table component in the ui-kit is also not the same as the one in the designs

Reply:
Pushed the changes